src/motor-interface-constants.h

```diff
@@ -95,6 +95,12 @@
 #define PIN_P8_19  (GPIO_0_BASE | 22)  // EHRPWM2A
 #define PIN_P8_26  (GPIO_1_BASE | 29)
 
+#define PIN_P8_43  (GPIO_2_BASE | 8) //Check http://www.ofitselfso.com/BeagleNotes/pinmux.pdf, select the GPIO device and the number
```

Comment:
Is the URL a relevant information that we need in particular for PIN_P8_43 ? I think the general documentation about GPIO things should be at a different place.